words/words.md

```diff
@@ -32,11 +32,11 @@
 
 <!--Sure, don't hoard toilet paper – but if policymakers fear fear itself, they'll downplay real dangers to avoid "mass panic". Fear's not the problem, it's how we *channel* our fear. Fear gives us energy to deal with dangers now, and prepare for dangers later.-->
 
-Certo, non stare a fare scorte di carta igienica (o di pasta *NdT*) - ma se i politici hanno paura della paura, minimizzeranno i veri pericoli per evitare il "panico di massa".
+Certo, non stare a fare scorte di carta igienica (o di pasta *NdT*) - ma se i decisori politici hanno paura della paura stessa, minimizzeranno i veri pericoli per evitare il "panico di massa". Il problema non è la paura in sé, è come *incanaliamo* la nostra paura. La paura ci dà le energie per gestire i pericoli attuali e prepararci a quelli futuri.
 
 <!--Honestly, we (Marcel, epidemiologist + Nicky, art/code) are worried. We bet you are, too! That's why we've channelled our fear into making these **playable simulations**, so that *you* can channel your fear into understanding:-->
 
-Onestamente, noi (Marcel, epidemiologo e Nicky, arte/codice) siamo preoccupati. E crediamo che lo sia anche tu! Ecco perché abbiamo incanalato la nostra paura nel fare queste **simulazioni giocabili**, in modo che *tu* possa incanalare la tua paura verso la comprensione.
+Onestamente, noi (Marcel, epidemiologo e Nicky, artista/programmatore) siamo preoccupati, e crediamo che lo sia anche tu! Ecco perché abbiamo incanalato la nostra paura nel fare queste **simulazioni giocabili**, in modo che *tu* possa incanalare la tua paura verso la comprensione.
```

Comment:
we should probaly change also in the header arte/codice -> artista/programmatore.

Reply:
Ah, by the way, we should probably use "programmatrice" instead of "programmatore" for Nicky.

Reply:
> Ah, by the way, we should probably use "programmatrice" instead of "programmatore" for Nicky.

Good point! I think hen¹ is nonbinary, the best would be asking, unless you are sure.

¹ Swedish for he or she, because Swedish is amazing at pronouns

Reply:
A good approach to gender neutral stuff in italian: https://blog.neocabgame.com/post/188394177391/translating-the-cyberpunk-future

Possibilities that could be used;
Programmator*
Programmatorə

P.S. Sorry I showed up out of nowhere just to comment this. Just arrived and I'm still trying to figure out if there's anything left I can help with.

Reply:
Well, this is already valuable help (+ a nice article to read)! And I had never stumbled upon the second possibility, which is _so much better than the first_ (every time I see "*" I can't help but think of mistakes or footnotes).
Anyway, if you're willing to help with a more technical part, we are (or at least I am) really struggling with #16

Reply:
Yes, thanks for chiming in! I realized after I posted the comment the situation was less straightforward than I though. Anyway, according to the [twitter profile](https://twitter.com/ncasenmare), Nicky says "any pronoun's fine🦄". Regarding other stuff to help with I raised #13 and I still think it can be useful.

Reply:
another option would be anyway to go half back to previous translation with "artista/codice". Original is "art/code" which was first translated to "arte/codice". I guess that @harisont proposed a change to be more uniform with "Marcel Salathe, epidemiologo" but the English original was also non uniform in the description of the two authors. I agree that "arte/codice" does not sound that great in Italian so I was ok with the change.

Reply:
Just to give a final opinion, for me both "artista/codice" and "artista/Programmatorə" are fine. You can pick whatever you like best!